### ISSUE TITLE
Rules for Arabic numerals in numbered lists

### ISSUE THREAD

**Issue by tomerm** [w3c/alreq#118]
![image](https://cloud.githubusercontent.com/assets/5329257/26284654/4640c278-3e49-11e7-86cf-08fc5c13c9b4.png)
Numbered lists were mentioned indirectly in the context of footnotes in #109 . However, they are not explicitly called out in #101 and https://github.com/w3c/alreq/wiki/Arabic-numerals-(Draft).

I wonder if there is some general guidance on type of Arabic numerals to be used as part of numbered lists. May be it is a function of country / text content / genre of literature / something else. 

Assuming we have a numbered list of 10 items. 

- overwhelming majority (8 items) include exclusively Arabic text, 
- one item includes a mixed Arabic and non Arabic text (for example, a sentence is in Arabic language  mentioning / quoting some concept in foreign language), 
- one item includes exclusively non Arabic text (for example name of foreign company in foreign language).

Would be a general preference to use Arabic-Indic numerals for such a list ? 
In other words, would next example be less preferable (as compared to first image above): 

![image](https://cloud.githubusercontent.com/assets/5329257/26284852/372000a2-3e4d-11e7-8187-209c64c13ef2.png)

Many thanks in advance. 

PS. In my humble opinion it would be definitely expected from the software supporting numbered list (such as rich text editor) to provide support for both Arabic-Indic and Arabic-European numerals. While overwhelming majority of rich text editors support Arabic-European numbered lists, only few support Arabic-Indic numbered lists.

**Comment by shervinafshar** [comment]
As far as I can say, your first screenshot with Hebrew and Arabic shows a redundant bullet (compare Hebrew with Arabic for the part of text coming after numbered item 4).

The second screenshot is legit for when the author of the document decides to use ASCII (what you called as "Arabic-European") numerals as that ASCII numerals are de facto to be used with Arabic text in some Northern African countries and used commonly (either intentionally or unintentionally) in other Arabic-writing locales.

If you are interested in drafting some text for [3.8](https://w3c.github.io/alreq/#h_numbers) on numbered lists, you're more than welcome to put together your research and submit a draft.

**Comment by tomerm** [comment]
@shervinafshar thanks a lot for the comment and suggestion. In https://github.com/w3c/alreq domain I see myself as facilitator / participant / observer. I plan to contribute to a similar document / working group https://github.com/w3c/hlreq focusing on Hebrew text layout. 

Different numbering systems are used in different Arabic speaking countries. I believe the default is standardized in [CLDR](http://cldr.unicode.org/). Arabic numeric shaping (0123456789 vs. ٠ ١٢٣٤٥٦٧٨٩ vs. ...) - appears as part of "Numbering systems" -http://cldr.unicode.org/translation/numbering-systems . For example http://www.unicode.org/repos/cldr/trunk/common/bcp47/number.xml or https://github.com/unicode-cldr/cldr-core/blob/master/supplemental/numberingSystems.json have
"٠ ١٢٣٤٥٦٧٨٩" defined as "arab" and "1234567890" defined as "latn". Mapping to locale is done via property **defaultNumberingSystem** which is "arab" for ar (http://unicode.org/cldr/trac/browser/trunk/common/main/ar.xml) but "latn" for ar_MA (http://unicode.org/cldr/trac/browser/trunk/common/main/ar_MA.xml) and ar_DZ (http://unicode.org/cldr/trac/browser/trunk/common/main/ar_DZ.xml)

My goal is to assure proper / optimal user experience in software products. This includes providing  necessary flexibility / control to assure requirements  of Arabic speaking users are addressed. Based on your comments it sounds that it is definitely necessary to be able to author numbered list using both Western Arabic (aka Arabic-European) and Eastern Arabic (aka Arabic-Indic) numerals. It boils down that in different countries (see CLDR above) and under different technological limitations (see your comment on UTF-8 vs ASCII) the default numbering system to be used can be either Western Arabic or Eastern Arabic. However in general it is really up to the author to decide which one better fits his/her needs. And from software perspective both options (Western Arabic or Eastern Arabic) should be available. Please let me know if this conclusion is inaccurate. 

_"redundant bullet"_ is actually a star (*) used in Arabic part as a footnote reference. Not sure why it does not appear in Hebrew one.
